pyproject.toml

```
@@ -40,7 +40,7 @@ dependencies = [
     "requests",
     "scipy",
     "torch>=2.2.0",
-    "transformers>=4.47.0,<4.48.0",
+    "transformers>=4.49.0",
```

Let's add an upper bound for safety and to follow what we had before!

```suggestion
    "transformers>=4.49.0,<4.50.0",
```